### ISSUE TITLE
training: unoperating learning rate

### ISSUE THREAD

**Issue by delhomer** [Oslandia/mapillary_classification#60]
In the current state of the code, the learning rate is totally useless: the learning rate is considered as an parameter of the `ConvolutionalNeuralNetwork` class constructor, however it is not an attribute of this class.

In the new Keras-based API, the learning rate must be defined with the optimizer function, as follows (for instance):
```
opt = Adam(learning_rate=0.001, decay=0.0001)
model.compile(loss='binary_crossentropy', optimizer=opt, metrics=['acc'])
```
